crates/compression/src/eviction_policy.rs

```diff
@@ -7,14 +7,19 @@ use crate::ports::EvictorDb;
 /// Evictor for a single keyspace
 #[derive(Debug)]
 pub(crate) struct CacheEvictor<T> {
-    pub keyspace: std::marker::PhantomData<T>,
     /// Set of keys that must not be evicted
-    pub keep_keys: HashSet<RegistryKey>,
+    keep_keys: HashSet<RegistryKey>,
+    _marker: std::marker::PhantomData<T>,
```

Comment:
The type already says "marker", I'd rather describe what it does.

```suggestion
    _keyspace_marker: std::marker::PhantomData<T>,
```